aspects/request_timeout_handler_test.go

```diff
@@ -0,0 +1,90 @@
+package aspects
+
+import (
+	"github.com/julienschmidt/httprouter"
+	"github.com/prebid/prebid-server/config"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+const reqTimeInQueueHeaderName = "X-Ngx-Request-Time"
+const reqTimeoutHeaderName = "X-Ngx-Request-Timeout"
+
+func TestQueuedRequestTimeoutWithTimeout(t *testing.T) {
```

Comment:
Consider a table driven test (ex: enforcement_test.go) since your tests are already in a similar format. It would be easier to understand what the test parameter inputs to `ExecuteAspectRequest`.

Reply:
Done!